packages/nextjs/components/scaffold-eth/Faucet.tsx

```diff
@@ -82,49 +84,42 @@ export const Faucet = () => {
 
   return (
     <div>
-      <label htmlFor="faucet-modal" className="btn btn-primary btn-sm px-2 rounded-full font-normal normal-case">
+      <Trigger id="faucet-modal" className="btn btn-primary btn-sm px-2 rounded-full font-normal normal-case">
```

Comment:
and  it's better to move id to constant outside of component and use it in both trigger and modal